frontend/app/views/bundle/bundlesLanding.scala.html

```diff
@@ -109,19 +95,14 @@ <h2 class="bundles__subheading"><span class="bundles__wrap-line">Become a </span
 									<p>Choose the package you want: Everyday+, Sixday+, Weekend+ and Sunday+</p>
 								</li>
 								<li class="js-digital-benefits">
-									<h3>Digital</h3>
-									<p>All the benefits of the digital subscription</p>
+									<h3>All the benefits of a Digital subscription</h3>
```

Comment:
Why does Digital have a capital letter here when it doesn't in the section above?